README.md

```diff
@@ -1601,8 +1615,242 @@ p.createFeeRule(data: {
     currency: string;
   }>;
 })
+
+```
+### Payment Request
+
+Instanitiate Payment Request using constructor that has been injected with Xendit keys
+
+```js
+const { PaymentRequest } = x;
+const r = new PaymentRequest();
+```
+
+Example: Create a Payment Request
+
+```js
+r.createPaymentRequest({
+      amount: 1500,
+      currency: 'PHP',
+      payment_method: {
+        type: 'EWALLET',
+        ewallet: {
+          channel_code: 'GRABPAY',
+          channel_properties: {
+            success_return_url: 'https://redirect.me/goodstuff',
+            failure_return_url: 'https://redirect.me/badstuff',
+          },
+        },
+        reusability: 'ONE_TIME_USE',
+      }
+}).then(({ id }) => {
+  console.log(`payment request created with ID: ${id}`);
+});
+```
+
+
+#### Create payment request
+
+```ts
+r.createPaymentRequest(data: {
+    currency: PaymentRequestCurrencies;
+    amount: number;
+    reference_id?: string;
+    customer_id?: string;
+    country: PaymentRequestCountries;
+    description?: string;
+    payment_method: object;
+    channel_properties?: PaymentRequestChannelProperties;
+    metadata?: object;
+    idempotencty_key?: string;
+    for_user_id?: string;
+})
+```
+
+#### List payment requests
+
+```ts
+r.listpaymentrequests(data: {
+    id?: string;
+    reference_id?: string;
+    customer_id?: string;
+    type?: PaymentRequestType;
+    channel_code?: string;
+    status?: PaymentRequestStatuses;
+    limit?: number;
+    after_id?: string;
+    before_id?: string;
+    for_user_id?: string;
+})
+```
+
+#### Get payment request details by ID
+
+```ts
+r.getPaymentRequestByID(data: {
+    id: string;
+    for_user_id?: string;
+})
+```
+
+#### confirm payment request
+
+```ts
+r.confirmPaymentRequest(data: {
+    id: string;
+    auth_code: string;
+    idempotencty_key?: string;
+    for_user_id?: string;
+})
+```
+
+#### resend payment request
```

Comment:
can this say "resend auth for payment request`

Reply:
yeah, updated the function name